tests/pyfunc/test_docker.py

```diff
@@ -0,0 +1,136 @@
+import difflib
+import os
+import shutil
+from unittest import mock
+
+import pytest
+import sklearn
+import sklearn.neighbors
+
+import mlflow
+from mlflow.models.flavor_backend_registry import get_flavor_backend
+from mlflow.utils.env_manager import CONDA, VIRTUALENV
+from mlflow.version import VERSION
+
+_MLFLOW_ROOT = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", ".."))
+_RESOURCE_DIR = os.path.join(_MLFLOW_ROOT, "tests", "resources", "dockerfile")
+
+
+def log_model():
+    with mlflow.start_run() as run:
+        knn_model = sklearn.neighbors.KNeighborsClassifier()
+        model_info = mlflow.sklearn.log_model(
+            knn_model,
+            "model",
+            pip_requirements=[
+                f"mlflow=={mlflow.__version__}",
+                f"scikit-learn=={sklearn.__version__}",
+            ],  # Skip requirements inference for speed up
+        )
+        return model_info.model_uri
+
+
+class TestParam:
```

Comment:
Can we rename this class? pytest thinks this is a test and emits a warning like this:

```
  /home/runner/work/mlflow/mlflow/tests/pyfunc/test_docker.py:33: PytestCollectionWarning: cannot collect test class 'TestParam' because it has a __init__ constructor (from: tests/pyfunc/test_docker.py)
    class TestParam:
```

Reply:
```suggestion
class _Param:
```

?